### PR TITLE
zh-CN: fix incorrect styling

### PR DESCRIPTION
**Pull request by PassionPenguin**
### Description

Fix incorrect bold styling

### Motivation

the styling is invalid

### Related issues and pull requests

PTAL @JasonLamv-t  :)

### CODE REVIEW

**Comment by PassionPenguin** [comment]
plus, translating documents should be faithful to the original ones, and should not have this remark:

Screenshot 2024-04-07 at 10 22 46 AM

File: files/zh-cn/web/css/calc/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ width: calc(100% - 80px);
 
 ### 备注
 
-- **`+` 和 `-` 运算符的两边必须要有空白字符。**比如，`calc(50% -8px)` 会被解析成为一个无效的表达式，解析结果是：一个百分比 后跟一个负数长度值。而加有空白字符的、有效的表达式 `calc(8px + -50%)` 会被解析成为：一个长度 后跟一个加号 再跟一个负百分比。
+- **`+` 和 `-` 运算符的两边必须要有空白字符。** 比如，`calc(50% -8px)` 会被解析成为一个无效的表达式，解析结果是：一个百分比 后跟一个负数长度值。而加有空白字符的、有效的表达式 `calc(8px + -50%)` 会被解析成为：一个长度 后跟一个加号 再跟一个负百分比。
```
Comment (yin1999) [inline files/zh-cn/web/css/calc/index.md:32]: minor (see: https://github.com/mdn/content/blob/b3e92f9e0724a177977560b51673b391bfdfc5e4/files/en-us/web/css/calc/index.md?plain=1#L41):

```suggestion
- `+` 和 `-` 运算符的**两边必须要有{{Glossary("whitespace", "空白字符")}}**。比如，`calc(50% -8px)` 会被解析成为一个无效的表达式，解析结果是：一个百分比 后跟一个负数长度值。而加有空白字符的、有效的表达式 `calc(8px + -50%)` 会被解析成为：一个长度 后跟一个加号 再跟一个负百分比。
```

**Comment by yin1999** [comment]
> plus, translating documents should be faithful to the original ones, and should not have this remark:
> 
> Screenshot 2024-04-07 at 10 22 46 AM
> if the translator thinks, that this adding is necessary, then should he goes to contribute to the original ones, than modifying the translation.

Feel free to remove this :)

File: files/zh-cn/web/css/calc/index.md (hunk 1 above)
Comment (PassionPenguin) [reply files/zh-cn/web/css/calc/index.md]: yeap, didn't check the source first...thx!

**Comment by PassionPenguin** [comment]
> > plus, translating documents should be faithful to the original ones, and should not have this remark:
> > Screenshot 2024-04-07 at 10 22 46 AM
> > if the translator thinks, that this adding is necessary, then should he goes to contribute to the original ones, than modifying the translation.
> 
> Feel free to remove this :)

that should be, if possible, in another pr (unrelated)

**Review by yin1999 — APPROVED** [review]
Thanks. Actually, there is a lot of content in the document that does not comply with [our standards](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md) (issues left over from [kuma](https://wiki.mozilla.org/MDN/Archives/Kuma)). It is best to clean up the entire document.

**Comment by PassionPenguin** [comment]
> Thanks. Actually, there is a lot of content in the document that does not comply with [our standards](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md) (issues left over from [kuma](https://wiki.mozilla.org/MDN/Archives/Kuma)). It is best to clean up the entire document.

hi yin1999，sorry to bother you，but it would be nice if you could provide some suggestion on improving and contributing to the project. Shall I know where can find documents that need to be translated/improved…? Appreciations for your reading and answering!

**Comment by yin1999** [comment]
> > Thanks. Actually, there is a lot of content in the document that does not comply with [our standards](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md) (issues left over from [kuma](https://wiki.mozilla.org/MDN/Archives/Kuma)). It is best to clean up the entire document.
> 
> hi yin1999，sorry to bother you，but it would be nice if you could provide some suggestion on improving and contributing to the project. Shall I know where can find documents that need to be translated/improved…? Appreciations for your reading and answering!

Firstly, you may find some issues that you can help with: [Issues with `l10n-zh` label](https://github.com/mdn/translated-content/issues?q=is%3Aopen+is%3Aissue+label%3Al10n-zh), especially [those with the "good first issue" label](https://github.com/mdn/translated-content/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+label%3Al10n-zh).

Besides, we recommend reading the [localization guidelines](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md). We provide a lot of information including formatting in the article, which can help you know what should be revised and how we can contribute. If you want to help us provide more localized documentation, the [glossary](https://developer.mozilla.org/zh-CN/docs/Glossary) is a great place to start, there are many documents that have not been translated (check the links in the sidebar, you may find many untranslated pages).

**Comment by PassionPenguin** [comment]
> > > Thanks. Actually, there is a lot of content in the document that does not comply with [our standards](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md) (issues left over from [kuma](https://wiki.mozilla.org/MDN/Archives/Kuma)). It is best to clean up the entire document.
> > 
> > 
> > hi yin1999，sorry to bother you，but it would be nice if you could provide some suggestion on improving and contributing to the project. Shall I know where can find documents that need to be translated/improved…? Appreciations for your reading and answering!
> 
> Firstly, you may find some issues that you can help with: [Issues with `l10n-zh` label](https://github.com/mdn/translated-content/issues?q=is%3Aopen+is%3Aissue+label%3Al10n-zh), especially [those with the "good first issue" label](https://github.com/mdn/translated-content/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+label%3Al10n-zh).
> 
> Besides, we recommend reading the [localization guidelines](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md). We provide a lot of information including formatting in the article, which can help you know what should be revised and how we can contribute. If you want to help us provide more localized documentation, the [glossary](https://developer.mozilla.org/zh-CN/docs/Glossary) is a great place to start, there are many documents that have not been translated (check the links in the sidebar, you may find many untranslated pages).

that's a very nice and comprehensive guide! I will take a look!

**Comment by PassionPenguin** [comment]
hi, @yin1999, will it possible to have a pr containing some 1838 files....which got format problems? (complete by regex...)

**Comment by yin1999** [comment]
> hi, @yin1999, will it possible to have a pr containing some 1838 files....which got format problems? (complete by regex...)
> 
> if possible, check [PassionPenguin@973dc9d](https://github.com/PassionPenguin/translated-content/commit/973dc9d2fcec0240090f56c0f421bdff3855bee1)

You can do this. But please split it into multi-parts -- At most 200 files per PR, We recommend you to make PRs according to the folder structures

**Comment by PassionPenguin** [comment]
> > hi, @yin1999, will it possible to have a pr containing some 1838 files....which got format problems? (complete by regex...)
> > if possible, check [PassionPenguin@973dc9d](https://github.com/PassionPenguin/translated-content/commit/973dc9d2fcec0240090f56c0f421bdff3855bee1)
> 
> You can do this. But please split it into multi-parts -- At most 200 files per PR, We recommend you to make PRs according to the folder structures

well...i may be possible to complete the task within this holiday...big task...

(chrome not correctly taking up all cores, causing computer lag even just when opening the branch page...)

**Pull request merged**